src/Microsoft.Framework.Caching.SqlServer/SqlServerCacheExtensions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Caching.Distributed;
+using Microsoft.Framework.Caching.SqlServer;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection
+{
+    public static class SqlServerCacheExtensions
+    {
+        public static IServiceCollection AddSqlServerCache([NotNull] this IServiceCollection services)
+        {
+            services.AddOptions();
+            services.AddSingleton<IDistributedCache, SqlServerCache>();
```

Comment:
Should this be TryAddX? I guess it depends on whether it is valid to have multiple implementations of the interface registered at the same time.